app/src/main/java/com/futurice/freesound/common/utils/Parcels.kt

```diff
@@ -0,0 +1,92 @@
+
+package com.futurice.freesound.common.utils
+
+import android.os.Parcel
+import android.os.Parcelable
+import java.math.BigDecimal
+import java.math.BigInteger
+import java.util.*
+import kotlin.collections.ArrayList
+
+interface KParcelable : Parcelable {
```

Comment:
Also tests, or at least tests for the Parcel generation from the data model perspective which would exercise these.